src/components/SubNav/subNavChangelogData.ts

```diff
@@ -9,6 +9,13 @@
 import { ChangelogData } from "../../utils/ComponentChangelogTable";
 
 export const changelogData: ChangelogData[] = [
+  {
+    date: "Prerelease",
+    version: "Prerelease",
+    type: "Update",
+    affects: ["Functionality"],
+    notes: ["Adds `as` prop to `SubNavLink`."],
```

Comment:
Do you feel that using the `as` prop with `SubNavLink` is a pattern that would be applicable to any NYPL app that implements the `SubNav` component? If so, I think an example in the docs would be useful.

Reply:
Yes, I think that would be helpful. Thanks.